src/agent/doc/index.rst

```diff
@@ -558,4 +558,4 @@ useful when certain interactions shouldn't be influenced by the memory context::
 .. _`Structured Output with PHP class`: https://github.com/symfony/ai/blob/main/examples/openai/structured-output-math.php
 .. _`Structured Output with array`: https://github.com/symfony/ai/blob/main/examples/openai/structured-output-clock.php
 .. _`Chat with static memory`: https://github.com/symfony/ai/blob/main/examples/memory/static.php
-.. _`Chat with embedding search memory`: https://github.com/symfony/ai/blob/main/memory/mariadb.php
+.. _`Chat with embedding search memory`: https://github.com/symfony/ai/blob/main/examples/memory/embedding-search.php
```

Comment:
partially right, i'd say
```suggestion
.. _`Chat with embedding search memory`: https://github.com/symfony/ai/blob/main/examples/memory/mariadb.php
```

Reply:
I reverted the change for now and will fix the link in another PR, as other such stuff may be broken, so they are better fixed together if it is the case.

Reply:
Fixed [here](https://github.com/symfony/ai/pull/291).